packages/marko-web-deferred-script-loader/components/register.marko

```diff
@@ -16,10 +16,25 @@ $ const {
   },
 });
 
+$ const delayMs = parseInt(input.delayMs, 10) || undefined;
 $ const init = `function() { ${input.init || ''} }`;
 $ const initOnly = defaultValue(input.initOnly, false);
 $ const onScriptBuild = `function(script) { ${input.onScriptBuild || ''} }`;
 $ const onScriptLoad = `function() { ${input.onScriptLoad || ''} }`;
 $ const attrs = JSON.stringify(getAsObject(input, "attrs"));
 
-<script>deferScript('register', { name: '${input.name}', src: '${input.src}', on: '${on}', requestFrame: ${requestFrame}, targetTag: '${targetTag}', init: ${init}, initOnly: ${initOnly}, onScriptBuild: ${onScriptBuild}, onScriptLoad: ${onScriptLoad}, attrs: ${attrs} });</script>
+<script>
+  deferScript('register', {
```

Comment:
Change this back to one line... I realize it's harder to read, but it was left as one line for browser minification purposes (since marko does not automatically flatten `<script>` elements)